sips/sip-x/sip-x-nft-metadata.md

```diff
@@ -0,0 +1,389 @@
+            "items: {
+                "type": "object",
+                "required": ["value"],
+                "properties": {
+                    "display_type": "string",
+                    "trait_type": "string",
+                    "value": {"oneOf": [{"type": "object"}, {"type": "string"}, {"type": "number"}, {"type": "integer"}, {"type": "boolean"} {"type: "array"}},
+                }
+            }
+        },
+        "properties": {
+            "type": "object",
+            "description": "Arbitrary properties. Values may be strings, numbers, object or arrays."
+        },
+        "localization": {
+            "type": "object",
+            "required": ["uri", "default", "locales"],
+            "properties": {
+                "uri": {
+                    "type": "string",
+                    "description": "The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request. See section about localization for more rules"
+                },
+                "default": {
+                    "type": "string",
+                    "description": "The locale of the default data within the base JSON"
+                },
+                "locales": {
+                    "type": "array",
+                    "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+                }
+            }
+        },
+        "image_data": {
+            "type": "string",
+            "description": "Raw SVG image data. Deprecated. Use `properties.image_data`."
+        },
+        "external_url": {
+            "type": "string",
+            "description": "Url to view the item on a 3rd party web site. Deprecated. Use `properties.external_url`."
+        },
+        "animation_url": {
+            "type": "string",
+            "description": "URL to a multi-media attachment for the item. Deprecated. Use `properties.animation_url`."
+        },
+    }
+}
+```
+
+The lengths of string values is not restricted. Nowadays, clients should be smart enough to deal with values of different lengths.
+
+### Example
+
+token101.json
+
+```
+{
+  "version": 1,
+  "name": "Foo #101",
+  "image": "ipfs://somerandomecid",
+  "attributes": [
+     {
+      "trait_type": "hair",
+      "value": "red",
+    },
+    {
+      "trait_type": "strength",
+      "display_type": "number",
+      "value": 99,
+    },
+  ],
+  "properties": {
+      "collection":  "Foo Collection",
+      "collection_size":  "10000"
+  },
+  "localization": {
+      "uri": "ipfs://somerandomcid/{locale}.json",
+      "default": "en",
+      "locales": ["en", "pt-BR", "de"]
+  }
+}
+```
+
+de.json
+
+```
+{
+    "version": 1,
+    "attritbutes: [
```

Comment:
another minor nit: typo here :)

Reply:
and below

Reply:
fixed in 2bc81e987b43de79a4c91c16272947849e0d142c